### DISCUSSION TITLE
CMB revisited

### DISCUSSION THREAD

**Discussion by genotics** [a-cosmology-group/acg#222] [General]
Members may wish to refresh their memories with a little history.
<https://youtu.be/cBrjDNRDEwQ?si=UgBVT0tzwKfLXWpx>
-R

**Comment by HanDeBruijn** [comment]
Other recent examples of "Science by news release" as published at [nu.nl](https://www.nu.nl/wetenschap) in the Netherlands:

- [Lense–Thirring precession after a supermassive black hole disrupts a star](https://www.nature.com/articles/s41586-024-07433-w)
![een](https://github.com/a-cosmology-group/acg/assets/135433222/442a991c-26f1-4c27-8556-dd2761f082f0)
- [Continuum emission from within the plunging region of black hole discs](https://academic.oup.com/mnras/article/531/1/366/7671518?login=false)
![twee](https://github.com/a-cosmology-group/acg/assets/135433222/b37c0888-b3a3-406b-8f83-7bf84fad6d32)
- [GA-NIFS: JWST discovers an offset AGN 740 million years after the big bang](https://academic.oup.com/mnras/article/531/1/355/7671512?searchresult=1&login=false)
![drie](https://github.com/a-cosmology-group/acg/assets/135433222/88f87800-9aff-4642-97ac-80f884aceeb4)

**Comment by mikehelland** [comment]
Sure a lot of snark in that video.

One thing they don't say is what the CMB actually is.
